### PR TITLE
Return statement generated once v1

### PR DESCRIPTION
**Pull request by tanatana**
#1462

### CODE REVIEW

File: goagen/gen_main/generator.go [hunk 1]
```diff
@@ -384,6 +384,16 @@ func funcMap(appPkg string, actionImpls map[string]string) template.FuncMap {
 			}
 			return body
 		},
+		"printResp": func(name string) bool {
+			if actionImpls == nil {
```
[the remark this answers is not in the document]
Comment (tanatana) [reply goagen/gen_main/generator.go]: Thank you, I am going to fix it.
By the way, the same processing is on line 378. Is it better to modify it the same way?
https://github.com/goadesign/goa/pull/1464/files/a828f7fe2be77a77c26165802a16662d5fde966d#diff-34b00d9b7f5787450c4e36c8cedbd6b1R378

File: goagen/gen_main/generator.go [hunk 2]
```diff
@@ -375,19 +375,13 @@ func funcMap(appPkg string, actionImpls map[string]string) template.FuncMap {
...
 				return defaultActionBody
 			}
 			return body
 		},
 		"printResp": func(name string) bool {
-			if actionImpls == nil {
-				return true
-			}
 			_, ok := actionImpls[name]
 			if !ok {
```
Comment (raphael) [inline goagen/gen_main/generator.go]: You can also replace
```go
if !ok {
    return true
}
return false
```
with
```go
return !ok
```
:)

Comment (tanatana) [reply goagen/gen_main/generator.go]: 🙈

**Comment by raphael** [comment]
Thank you! one last request I promise :) could you port this change to `master` please? just branch off of master and cherry-pick this commit into it then submit a new PR. Thanks again!

**Pull request merged**